boards/arm/nrf52840_pca10090/nrf52840_pca10090_defconfig

```diff
@@ -7,6 +7,11 @@ CONFIG_BOARD_NRF52840_PCA10090=y
 # Enable MPU
 CONFIG_ARM_MPU=y
 
+# The GPIO driver is required by this board's initialization code (board.c),
```

Comment:
Remove from defconfig, it will get set by the select, so you don't need it here.

Reply:
@galak I intentionally commented out the `CONFIG_GPIO=y` line instead of removing it completely, and I added the comment explaining why `GPIO` is not enabled here. I wanted to leave a trace in case someone searched for `CONFIG_GPIO=y` in boards, for instance.
You want me to remove this commented out line? Or the whole comment?

Reply:
I think we should remove it.

Reply:
Removed.